### PR TITLE
Split documentation howto section into several files

### PR DESCRIPTION
**Pull request by ewjoachim**
Based on #62 

Also adds documentation pertaining to crons.

### CODE REVIEW

File: docs/discussions.rst [hunk 1]
```diff
@@ -129,7 +129,7 @@ Worker
     to launch jobs to execute your tasks.
 
 Job Store
-    The job store resposability is to store and retrieve jobs. In Procrastinate, the
+    The job store responsability is to store and retrieve jobs. In Procrastinate, the
```
Comment (pmourlanne) [inline docs/discussions.rst]:
```suggestion
    The job store responsibility is to store and retrieve jobs. In Procrastinate, the
```
Might as well :>

File: docs/howto/command_line.rst [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+    $ procrastinate --help
+
+or:
+
+.. code-block:: console
+
+    $ python -m procrastinate --help
+
+Please read the included help to get famailiar with its commands and parameters:
```
Comment (pmourlanne) [inline docs/howto/command_line.rst]:
```suggestion
Please read the included help to get familiar with its commands and parameters:
```

File: docs/howto/locks.rst [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+tasks will run sequentially and in order.
+
+A good string identifier for the lock is a string identifier of
+the shared resource, UUIDs are well suited for this.
+If multiple resources are implicated, a combination
+of their identifiers could be used (there's no hard
+limit on the length of a lock string, but stay reasonable).
+
+A task can only take a single lock so there's no dead-lock scenario possible
+where 2 running tasks are waiting one another.
```
Comment (pmourlanne) [inline docs/howto/locks.rst]:
```suggestion
where two running tasks are waiting one another.
```

File: docs/howto/worker.rst [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+.. code-block:: console
+
+    $ procrastinate --verbose --app=dotted.path.to.app worker [queue [...]]
+
+or, identically, use the code way::
+
+    app.run_worker(queues=["queue", ...])
+
+In both case, not specifying queues will tell Procrastinate to listen to every queue.
```
Comment (pmourlanne) [inline docs/howto/worker.rst]:
```suggestion
In both cases, not specifying queues will tell Procrastinate to listen to every queue.
```

File: procrastinate/cli.py [hunk 5]
```diff
@@ -0,0 +1,204 @@
...
+def defer(
+    app: procrastinate.App,
+    task: str,
+    json_args: str,
+    lock: Optional[str],
+    at: Optional[str],
+    _in: Optional[int],
+):
+    """
+    Creates a job from the given task, to be executed by a worker.
```
Comment (pmourlanne) [inline procrastinate/cli.py]:
```suggestion
    Create a job from the given task, to be executed by a worker.
```
For consistency?

File: procrastinate/tasks.py [hunk 6]
```diff
@@ -73,7 +75,44 @@ def configure(
         task_kwargs: Optional[types.JSONDict] = None,
         schedule_at: Optional[datetime.datetime] = None,
         schedule_in: Optional[Dict[str, int]] = None,
-    ) -> jobs.Job:
+        queue: Optional[str] = None,
+    ) -> jobs.JobLauncher:
+        """
+        Configures the job with all the specific settings, defining how the job
```
Comment (pmourlanne) [inline procrastinate/tasks.py]:
```suggestion
        Configure the job with all the specific settings, defining how the job
```
For consistency?

**Comment by ewjoachim** [comment]
Well, if your reviewed everything and you're OK, then I'm ok too :)

**Pull request merged**